src/main/scala/treadle/blackboxes/AsyncResetBlackBox.scala

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2020 The Regents of the University of California (Regents)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+
+package treadle.blackboxes
+
+import firrtl.ir.{Param, Type}
+import treadle._
+import treadle.executable._
+
+/**
+  * Implements a single bit register with asynchronous reset
+  */
+class AsyncResetReg(val instanceName: String) extends ScalaBlackBox {
```

Comment:
Is this necessary anymore? The module in Rocket Chip is no longer a BlackBox: https://github.com/chipsalliance/rocket-chip/blob/2cee947a326459349006ab5aeae8f95e925e3d43/src/main/scala/util/AsyncResetReg.scala

Reply:
Removed AsyncResetReg